wordpress-mobile/WordPress-Android
# Update and optimize Quick Start task card illustrations

This PR updates the Quick Start task card illustrations to use their latest iterations and also extract the colors used by them, so we can have a single illustration for both light and dark modes.

|Light|Dark|
|---|---|
|![light](https://user-images.githubusercontent.com/830056/109049984-ac7e8e80-76b7-11eb-8ec8-d11d4c4633ce.png)|![dark](https://user-images.githubusercontent.com/830056/109049997-b0121580-76b7-11eb-9e13-52ec67c63467.png)|

To test:

1. Turn on the Improved My Site flag and restart the app.
1. Go to the My Site screen.
1. Create a new site.
1. Notice the Quick Start task cards.
1. Make sure all tasks have the correct illustrations and that nothing looks off.
1. Repeat for light/dark mode.

PR submission checklist:
